### DISCUSSION TITLE
Just released version 2.6.0!

### DISCUSSION THREAD

**Discussion by calebporzio** [livewire/livewire#3718] [5. Announcements]
I just tagged the v2.6.0 release. This is the biggest release since probably version 2.0 itself.

Holding off on announcing on twitter and email and such until we know there aren't any kinks.

Please post here (or ideally submit PRs) if there are any breaking changes or hiccups so we can catch them early.

Check out the release notes for details: https://github.com/livewire/livewire/releases/tag/v2.6.0

Blog post coming to Laravel news soon.

**Comment by delta1186** [comment]
Missed this post and already posted here: https://github.com/livewire/livewire/discussions/3720

Sorry about that.

**Reply by calebporzio** [reply]
All good!
